docs/crocks/Arrow.md

```diff
@@ -0,0 +1,77 @@
+# Arrow
+
+`Arrow a b`
+
+--
+
+--
+
+```js
+--
+```
+
+`Arrow` exposes the following functions on the constructor and instance:
+
+| Constructor | Instance |
+|:---|:---|
+| [`empty`](#empty), [`type`](#type) | [`both`](#both), [`concat`](#concat), [`contramap`](#contramap), [`empty`](#empty), [`first`](#first), [`inspect`](#inspect), [`map`](#map), [`promap`](#promap), [`runWith`](#runWith), [`second`](#second), [`type`](#type), [`value`](#value) |
+
+## Constructor
+
+### empty
+
+`Arrow m => () -> m () b`
+
+### type
+
+`() -> String`
+
+## Instance
+
+### both
+
+`Arrow m, Pair p => m a b ~> () -> m (p a b) (p c d)`
+
+### concat
+
+`Arrow m => m a b ~> m a b -> m a b`
+
+### contramap
+
+`Arrow m => m a b ~> (b -> a) -> m c b`
```

Comment:
So `contramap` works on the input of the `Arrow`, should be:

`Arrow m => m a b ~> (c -> a) -> m c b`